### DISCUSSION TITLE
System Requirements

### DISCUSSION THREAD

**Discussion by ghost** [getnikola/nikola#3594] [Q&A]
Alright so I want to run a static server off of this computer with 1 gig of ram is that enough or do i need more

**Comment by Kwpolska** [comment]
Nikola should be able to run on a machine with 1 GB of RAM. A server (such as nginx) should work fine with that amount as well.

**Comment by ralsina** [comment]
We build a bunch of sites and serve them off a 512MB virtual server
(everything in getnikola.com)
